src/dotnet/commands/dotnet-sln/add/SlnAddParser.cs

```diff
@@ -15,6 +15,10 @@ public static class SlnAddParser
                            Accept.OneOrMoreArguments(o => CommonLocalizableStrings.SpecifyAtLeastOneProjectToAdd)
                                  .With(name: LocalizableStrings.AddProjectPathArgumentName,
                                        description: LocalizableStrings.AddProjectPathArgumentDescription),
+                           Create.Option(
+                               "--place-project-in-root",
```

Comment:
@KathleenDollard  Not to bike-shed this more, but now that I think about this a little bit, `dotnet sln add` supports multiple projects.  Rather than calling this `--place-projects-in-root`, perhaps we can simplify this to just `--add-to-root` or `--place-in-root`?

@

Reply:
Probably adding to bike-shedding here, but add and place seem to read redundantly to the the add verb. `dotnet sln add --in-root X` ?

Reply:
I like it.

Reply:
I would suggest `--in-root` or `--root` as well, but I'm still waiting for a final decision from @KathleenDollard.